src/view/channel-detail-mobile.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import { SplitBackground } from '../component/background';
+import MainContent from '../component/main-content';
+import { Header, Title } from '../component/header';
+import { DetailField } from '../component/field';
+import { Button, BackButton, ButtonText } from '../component/button';
+import Card from '../component/card';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  btnWrapper: {
+    flex: 1,
+    justifyContent: 'flex-end',
+  },
+  deleteBtn: {
+    marginTop: 15,
```

Comment:
```suggestion

```

Reply:
We need the margin on smaller screens so the button isn't crammed against the fields when scrolling.